components/camera/server.go

```diff
@@ -31,7 +31,7 @@ type serviceServer struct {
 // NewRPCServiceServer constructs an camera gRPC service server.
 // It is intentionally untyped to prevent use outside of tests.
 func NewRPCServiceServer(coll resource.APIResourceCollection[Camera]) interface{} {
-	logger := logging.NewLogger("camserver")
+	logger := logging.Global().Sublogger("camserver")
```

Comment:
Why are we making this change as part of modmanager connection management?

Reply:
We don't have to. But I was adding logs for debugging in this file and found the logger was "misconfigured". Happy to revert.